.env

```
@@ -30,7 +31,27 @@ OKP4_COOKIE_POLICY_URL='https://okp4.network/cookie-policy'
 OKP4_GITHUB_OKP4D_URL='https://github.com/okp4/okp4d'
 OKP4_GITHUB_CONTRACTS_URL='https://github.com/okp4/contracts'
 OKP4_GITHUB_AWESOME_URL='https://github.com/okp4/awesome'
+OKP4_GITHUB_OKP4D_RELEASES_URL='https://github.com/okp4/contracts/releases/tag/v2.0.0'
+OKP4_GITHUB_OBJECTARIUM_URL='https://github.com/okp4/contracts/tree/v2.0.0/contracts/okp4-objectarium'
+OKP4_GITHUB_COGNITARIUM_MESSAGES_URL='https://github.com/okp4/contracts/blob/v2.0.0/contracts/okp4-cognitarium/src/msg.rs'
+OKP4_GITHUB_LAW_STONE_URL='https://github.com/okp4/contracts/tree/v2.0.0/contracts/okp4-law-stone/examples/single-source'
+OKP4_GITHUB_PREDICATE_BLOCK_URL='https://github.com/okp4/okp4d/blob/main/x/logic/predicate/block.go'
```

```suggestion
OKP4_GITHUB_PREDICATE_BLOCK_URL='https://github.com/okp4/okp4d/blob/v4.1.0/x/logic/predicate/block.go'
```

The current version of the nemeton testnet is `v4.1.0`